src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+// import categoriesReducer from './categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+    // categories: categoriesReducer,
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure the categories are present in the store. as you may see how u have commented it out